.test-infra/dataproc/create_flink_cluster.sh

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env bash
+#    Licensed to the Apache Software Foundation (ASF) under one or more
+#    contributor license agreements.  See the NOTICE file distributed with
+#    this work for additional information regarding copyright ownership.
+#    The ASF licenses this file to You under the Apache License, Version 2.0
+#    (the "License"); you may not use this file except in compliance with
+#    the License.  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+#    Runs init actions for Docker, Portability framework (Beam) and Flink cluster
+#    and opens an SSH tunnel to connect with Flink easily and run Beam jobs.
+#
+#    Pass the following command line args to run this script:
+#    $1: Cluster name
+#    $2: GCS bucket url for Dataproc resources (init actions)
+#    $3: SDK Harness' images repository from which images will be downloaded
+#    $4: SDK Harness' images to pull on dataproc workers (python,java,go)
+#    $5: Url to Flink .tar archive to be installed on the cluster
+#    $6: Number of Flink workers
+#    $7: Number of Flink slots
```

Comment:
Flink slots per worker? (sorry, I don't know about these things..)

Reply:
ok - it's a good idea to clarify this better. Will do. 👍  

According to my knowledge, task slots are a way of telling how many tasks a worker accepts. Thanks to this a Task Manager (located in a worker) can evenly divide resources per slot providing some sort of isolation between them.